robot/Cumulus/tests/browser/batch_gift_entry/donation_for_contact_dont_select_match.robot

```diff
@@ -32,6 +32,7 @@ Dont select match for contact new donation with grid changes
     Populate Field By Placeholder    Search Contacts    &{contact}[FirstName] &{contact}[LastName]
     Click Link    &{contact}[FirstName] &{contact}[LastName]
     Page Should Contain Link    Review Donations
+    Click Element With Locator    bge.field-input    Donation Amount
```

Comment:
This add looks to involve BGE functionality.  I am not sure how this affects Payments / Schedule Payments / Write Off Payments UI changes.

Reply:
Yes not related to this test case, had a small fix for bge so pushing in same branch instead of new one